### ISSUE TITLE
Printable accordion content

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#5955]
## What happened?

When a user prints out a page, e.g. https://staging.va.gov/burials-memorials/eligibility/ the content in the collapsed accordion is not included in the printed content. The user would need to open _every_ accordion manually before printing (or saving to a PDF).

For accordions on form pages (review & submit pages), it would not be possible to include the content without explicitly opening each one manually because React doesn't render the content when the accordion is collapsed.

Slack discussion: https://dsva.slack.com/archives/C0NGDDXME/p1568148789060400

## Specs:

- Device: Any
- Browser: Any
- Test User: Public

## Steps to Reproduce

1. Go to any VA page with an accordion, e.g. https://staging.va.gov/burials-memorials/eligibility/
2. Open one or more accordion sections
3. Press <kbd>⌘</kbd> + <kbd>p</kbd> to open the print dialog.
4. Take note that the collapsed accordions only show the header text. None of the content.

![Screen Shot 2020-02-18 at 1 55 37 PM](https://user-images.githubusercontent.com/136959/74774738-e2de2e00-5259-11ea-96ea-cd1811fef6e4.png)

## Desired behavior

When printing (creating a PDF), Include all the content within collapsed accordions.

## Possible Fixes (optional)

Update the CSS used while printing to make the accordion content visible.

## Acceptance Criteria
- [ ] Collapsed accordion content is printed

**Comment by jenstrickland** [comment]
@1Copenut — I think this may be a design bug, in that support for printing hidden content wasn't considered, but am not sure. What labels / assignments would you recommend for this issue?

**Comment by bkjohnson** [comment]
Here's an example of what the web component accordion looks like when printed:

![Screenshot of a print preview with one accordion item opened and showing](https://user-images.githubusercontent.com/2008881/168859935-09f376f1-56ea-4b6b-a5c0-62f0a2f85e22.png)

**Comment by Mottie** [comment]
I make it a habit to not say "simple" anymore 😄

**Comment by Mottie** [comment]
Component library updated to 42.1.1, which includes print media CSS to print accordion content even if it is collapsed.
